envConfig/local.env

```diff
@@ -6,3 +6,5 @@ SIMORGH_MPULSE_API_KEY=JLSEX-R76DK-JXL3A-UB7YH-NPSEW
 LOG_DIR='log'
 LOG_LEVEL='debug'
 APP_ENV=local
+#Need to prefix SIMORGH_ in order to make this variable on the client
```

Comment:
I'd put this at the top of each .env file and change it to

`# NB only variables prefixed with SIMORGH_ are available on both the server and client`

Reply:
Good shout, have updated